doc/adoc/user_guide.adoc

```diff
@@ -157,11 +157,19 @@ manually. Once done, the upgrade process uses `zypper dup` and expects
 all required repositories to be setup correctly.
 
 Migration product target::
-If set, the migration will use that product as a target.
+The default product target is SLES-15-SP1. The migration_product setting can be used
```

Comment:
This is not correct DMS determines the product

https://github.com/SUSE/suse-migration-services/blob/master/suse_migration_services/migration_config.py#L87

We can say that the default SP target is `SP1` but we cannot state that the default target is `SLES`

Reply:
Thanks I will reword that as

"The default product target will be the 15-SP1 release corresponding to the current product."

Does the read better?